frontend/src/lib/components/LemonCalendar/LemonCalendar.scss

```diff
@@ -0,0 +1,15 @@
+.LemonCalendar {
+    .LemonCalendar__today {
+        position: relative;
+        &:after {
+            position: absolute;
+            bottom: 5px;
+            left: calc(50% - 2px);
```

Comment:
The `2px` seems a bit magical and may get out of sync with width/height. Since they're all closely linked, there could be a variable `--lemon-calendar-today-radius` to make this more obvious (and then width/height is `calc(var(--lemon-calendar-today-radius) * 2)`